src/layout/GenericComponent.tsx

```diff
@@ -287,7 +252,14 @@ export function GenericComponent<Type extends ComponentExceptGroupAndSummary>(
     label: RenderLabel,
     legend: RenderLegend,
     componentValidations,
-    ...props,
+  };
+
+  const componentProps = {
+    ...fixedComponentProps,
+    // TODO: Pass on the node object instead of all the properties in it. This could work fairly simply for most
```

Comment:
Should we create a technical debt issue and put it on the backlog to ensure this will be fixed and not continues as a for every living todo? 🤔 :D

Reply:
Yup, I agree! Will do that.

Reply:
Ah, sorry, was about to create that issue, but it seems I did not properly read which TODO this was. I already have a working fix for this, in relation to the rest of the work I'm doing in #377. Fixing this would add ~1000 extra lines of changes to this PR, so I kept it out of this PR to keep it "small". No need for an issue, in other words.. 😆